lib/catalog/approval_transition.rb

```diff
@@ -39,28 +37,28 @@ def state_transitions
 
     def submit_order
       finalize_order
-      @order_item.update_message("info", "Submitting Order #{@order_item.order_id} for provisioning ")
```

Comment:
We can add `order_item.update_message("info", "Submitting Order Item for provisioning")` in `SubmitOrderItem#process`

Reply:
Added